src/MovieCards.jsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import './MovieCards.css'
+import { useStore } from "zustand";
+import { likedMoviesStore, searchStore } from "./App";
+
+
+export function MovieCard(props) {
+    // console.log("FROM movie card")
+    // console.log(props.data)
+    let [info, setInfo] = useState({})
+    let [liked, setLike] = useState(false)
+    
+
+    useEffect(() => {
+
+        (async () => { const url = `https://www.omdbapi.com/?i=` + props.data.imdbID + `&apikey=881763cb`
+            try {
+                const response = await fetch(url)
+                if (!response.ok) {
+                    throw new Error(`Response status: ${response.status}`)
+                }
+                info = await response.json()
+                //console.log(info)
+                setInfo(info)
+            } catch (error) {
+                console.error(error.message)
+            }})()
+    },[])
+
+
+    const addLike = likedMoviesStore((state) => state.addLike)
```

Comment:
your code will be much easier to read if you define all state variable in the start